particles/Chooser/Chooser.js

```diff
@@ -102,7 +102,7 @@ ${productStyles}
     }
     _render(props, state) {
       return {
-        person: 'Claire',
+        choices: '{{choices.description}}',
```

Comment:
Mustache (`{{...}}`) expression replacing is already a core capability of the templating engine, I'd like to see if we can leverage that system instead of recreating it. If we can come up with a way of getting the description information into the Particle as a data-model, then the Particle can include that data into the render-model (return value of `_render()`) and let the template engine interpolate the content.